bindata/network/openshift-sdn/001-crd.yaml

```diff
@@ -47,6 +47,10 @@ spec:
           type: integer
           minimum: 1
           maximum: 65535
+        mtu:
+          type: integer
+          minimum: 576
+          maximum: 65536
```

Comment:
65536 is 50 or 100 too large.

Reply:
This is openshift-sdn specific, so it would be 50, not 100. Except also isn't it 51 too large? It seems like the max MTU should be 65535, not 65536. But anyway, "576" and "65536" match the check in `validateOpenShiftSDN()`; we accept any value here that the code will accept there.